process/smartContract/process.go

```diff
@@ -540,7 +540,8 @@ func (sc *scProcessor) DeploySmartContract(
 
 func (sc *scProcessor) printScDeployed(vmOutput *vmcommon.VMOutput, tx data.TransactionHandler) {
 	scGenerated := make([]string, 0, len(vmOutput.OutputAccounts))
-	for addr := range vmOutput.OutputAccounts {
+	for key := range vmOutput.OutputAccounts {
```

Comment:
suggestion to change to:
```GO
for _, account := range vmOutput.OutputAccounts {
    if check.IfNil(account){
        continue
    }
    ......
```
:1st_place_medal: for finding this

Reply:
Changed to check `account == nil`.